src/article/metadata/ContributorIdentifierEditor.js

```diff
@@ -0,0 +1,62 @@
+import { Component, domHelpers } from "substance";
+import { createNodePropertyModels, createValueModel } from "../../kit/model";
+import DefaultNodeComponent from "../components/DefaultNodeComponent";
+import { ValueComponent } from "../../kit/ui";
+
+export default class ContributorIdentifierEditor extends ValueComponent {
+
+  dispose() {
+    this.context.editorState.removeObserver(this);
+  }
+
+  render($$) {
+    const node = this.props.model.getItems()[0];
+    const nodeId = node.id;
+    const el = $$('div')
+      .addClass(this._getClassNames())
+      .attr('data-id', nodeId);
+
+    el.append($$(ContributorIDEditorComponent, { node }).ref(node.id));
+
+    el.on('click', this._onClick);
+    return el;
+  }
+
+  _getClassNames() {
+    return 'sc-card sm-contributor-id';
+  }
+
+  _onClick(e) {
+    domHelpers.stopAndPrevent(e);
+  }
+}
+
+class ContributorIDEditorComponent extends DefaultNodeComponent {
+  _getClassNames() {
+    return 'sc-reference sm-metadata';
+  }
+
+  _createPropertyModels() {
```

Comment:
Looking at this, do you need it? I don't think you need the 'stringDate' part, hence you can just always return the createValueModel() part, in which case is the default implementation not enough? I.e. if you delete the function does it still work the same?

Reply:
Yeah, sorry. It was meant to be deleted but was left out by mistake